test/docker/keycloak-test-appliance/pom.xml

```diff
@@ -0,0 +1,95 @@
+<!--
+
+    Copyright (C) 2017 Red Hat, Inc. (jdcasey@commonjava.org)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.commonjava.indy.docker</groupId>
+    <artifactId>indy-test-docker-appliances</artifactId>
+    <version>1.2.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>indy-docker-keycloak-test-appliance</artifactId>
+  <packaging>docker</packaging>
+
+  <name>Indy :: Docker Images :: Keycloak Test Appliance Image</name>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <configuration>
+          <images>
+            <image>
+              <name>${docker-appliance-keycloak}</name>
+              <alias>keycloak-test-appliance</alias>
+              <build>
```

Comment:
This build section isn't what we want here. Everything else in this PR looks really great, but this part will re-build the Keycloak Docker image, when the module in test/docker/keycloak already did that. Here, we just want to use the image from the local Docker registry and run it.

I think literally the only change I'd make is to remove this build section. Thanks for figuring all of this stuff out!